barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplonk.hpp

```diff
@@ -285,24 +285,9 @@ template <typename Curve> class ShplonkVerifier_ {
         std::vector<Fr> inverted_denominators;
         inverted_denominators.reserve(num_gemini_claims);
         inverted_denominators.emplace_back((shplonk_eval_challenge - gemini_eval_challenge_powers[0]).invert());
-        size_t i = 0;
         for (const auto& gemini_eval_challenge_power : gemini_eval_challenge_powers) {
-            bool is_dummy_round = i > num_gemini_claims;
             Fr round_inverted_denominator = (shplonk_eval_challenge + gemini_eval_challenge_power).invert();
```

Comment:
because of the way these are used (in a multiplication with fold evaluations, which will be 0 for dummy rounds) it doesnt matter what values they take